crates/sui-core/src/quorum_driver/reconfig_observer.rs

```diff
@@ -75,28 +65,19 @@ impl ReconfigObserver<NetworkAuthorityClient> for OnsiteReconfigObserver {
     }
 
     async fn run(&mut self, quorum_driver: Arc<QuorumDriver<NetworkAuthorityClient>>) {
-        // A tiny optimization: when a very stale node just starts, the
-        // channel may fill up committees quickly. Here we skip directly to
-        // the last known committee by looking at SuiSystemState.
-        let authority_agg = self.create_authority_aggregator_from_system_state().await;
```

Comment:
This optimization is probably no longer necessary, and removing it allows us to get rid of the last pice of the system object load call.